x-pack/legacy/plugins/siem/public/pages/detection_engine/components/signals_histogram_panel/config.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as i18n from './translations';
+import { SignalsHistogramOption } from './types';
+
+export const signalsHistogramOptions: SignalsHistogramOption[] = [
+  { text: i18n.STACK_BY_RISK_SCORES, value: 'signal.rule.risk_score' },
+  { text: i18n.STACK_BY_SEVERITIES, value: 'signal.rule.severity' },
+  { text: i18n.STACK_BY_DESTINATION_IPS, value: 'destination.ip' },
+  { text: i18n.STACK_BY_ACTIONS, value: 'event.action' },
+  { text: i18n.STACK_BY_CATEGORIES, value: 'event.category' },
+  { text: i18n.STACK_BY_HOST_NAMES, value: 'host.name' },
+  { text: i18n.STACK_BY_RULE_TYPES, value: 'signal.rule.type' },
+  { text: i18n.STACK_BY_RULE_NAMES, value: 'signal.rule.name' },
+  { text: i18n.STACK_BY_SOURCE_IPS, value: 'source.ip' },
+  { text: i18n.STACK_BY_USERS, value: 'user.name' },
```

Comment:
nit: `STACK_BY_USER_NAMES`?

Reply:
For the i18n var, or the underlying text itself?

[Design docs](https://www.figma.com/file/aH3NX5VLIzkKUAuTOLet0i/%5BAwaiting-Dev%5D-Detection-Engine---MVP?node-id=179%3A81142) have it as `Top users`. Thoughts @MichaelMarcialis? Either way, looks like I missed the `Top` prefix for most of these, so thanks for bring attention to this @FrankHassanabad!